### PR TITLE
Add context menu option to open the user's channel from a usercard

### PR DESCRIPTION
**Pull request by Mm2PL**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

# Description

Adds an option to the menu after right clicking the avatar to open the channel in a new popup window. Not exactly the best UX but works. Ported as-is from Dankerino pajaDent

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -45,6 +45,7 @@
 - Minor: Add search to emote popup. (#3404)
 - Minor: Messages can now be highlighted by subscriber or founder badges. (#3445)
 - Minor: Add workaround for multipart emoji as described in [the RFC](https://mm2pl.github.io/emoji_rfc.pdf). (#3469)
+- Minor: Add context menu option to open the user's channel from a Usercard. (#3486)
```
Comment (zneix) [inline CHANGELOG.md:48]: I don't think this is the best wording, but I feel like it's easier to understand for an average user.
```suggestion
- Minor: Added a way to open channel popup by right-clicking avatar in a usercard. (#3486)
```

Comment (Felanbird) [reply CHANGELOG.md]: `the` avatar

File: CHANGELOG.md [hunk 2]
```diff
@@ -45,6 +45,7 @@
 - Minor: Add search to emote popup. (#3404)
 - Minor: Messages can now be highlighted by subscriber or founder badges. (#3445)
 - Minor: Add workaround for multipart emoji as described in [the RFC](https://mm2pl.github.io/emoji_rfc.pdf). (#3469)
+- Minor: Added a way to open channel popup by right-clicking avatar in a usercard. (#3486)
```
Comment (Mm2PL) [inline CHANGELOG.md:48]:
```suggestion
- Minor: Added a way to open channel popup by right-clicking the avatar in a usercard. (#3486)
```

**Review by Felanbird — APPROVED** [review]
Functionality wise I've used this in Dankerino before and I do think it could be useful, once and a while I wouldn't mind quickly hopping into someone's chat in this manor.

**Review by pajlada — APPROVED** [review]
Looks good to me, will sort out conflicts and merge in after #3566

**Pull request merged**